### PR TITLE
Replace Weather Underground API integration with Dark Sky API

### PR DESCRIPTION
**Pull request by srt4**
The Weather Underground API has been turned off: https://apicommunity.wunderground.com/weatherapi/topics/end-of-service-for-the-weather-underground-api

### CODE REVIEW

**Review by jonatan-ivanov — CHANGES_REQUESTED** [review]
How about

`String emoji = ":${mapIconToSlackIcon(new String(condition?.icon?:"unknown"))}:"`

File: slack-weather.groovy [hunk 1]
```diff
@@ -27,40 +29,31 @@ catch (Exception e) {
...
+String emoji = ":" +mapIconToSlackIcon(new String(condition?.icon?:"unknown"))  + ":"
 
 def slackRS = setSlackStatus(statusText, emoji, slackUserToken)
 println slackRS.statusLine
 
-static def getCondition(String wundergroundApiKey, String locationFormatted) {
-    def wundergroundRS = new RESTClient('http://api.wunderground.com').get(
-        path: "/api/$wundergroundApiKey/conditions/q/${locationFormatted}.json"
+def getCondition(String darkSkyApiKey, String latitude, String longitude) {
+    def conditionRs = new RESTClient('https://api.darksky.net/').get(
```
Comment (jonatan-ivanov) [inline slack-weather.groovy:49]: How about 
def conditionRs = new RESTClient('https://api.darksky.net/').get(path: "/forecast/$darkSkyApiKey/$latitude,$longitude")

**Review by jonatan-ivanov — CHANGES_REQUESTED** [review]

File: slack-weather.groovy (hunk 1 above)
Comment (srt4) [reply slack-weather.groovy:49]: Done

**Comment by srt4** [comment]
> How about
> 
> `String emoji = ":${mapIconToSlackIcon(new String(condition?.icon?:"unknown"))}:"`

Done

**Pull request merged**